src/main/java/uk/gov/hmcts/ccd/data/casedetails/search/SortOrderQueryBuilder.java

```diff
@@ -4,26 +4,34 @@
 import org.springframework.stereotype.Component;
 import uk.gov.hmcts.ccd.data.casedetails.search.MetaData.CaseField;
 
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 @Component
 public class SortOrderQueryBuilder {
 
     private static final String DATA_FIELD = "data";
     private static final String CREATED_DATE = "created_date";
     private static final String SPACE = " ";
     private static final String COMMA = ",";
+    private static final String SPECIAL_CHARS_REGEXP = "[\\s\\\\\\/,;\\)\\('\"`]";
+    private static final Pattern SPECIAL_CHARS_PATTERN = Pattern.compile(SPECIAL_CHARS_REGEXP);
 
     public String buildSortOrderClause(MetaData metaData) {
         StringBuilder sb = new StringBuilder();
         metaData.getSortOrderFields().forEach(sortOrderField -> {
-            if (sortOrderField.isMetadata()) {
-                sb.append(getMataFieldName(sortOrderField.getCaseFieldId()));
-            } else {
-                sb.append(convertFieldNameToJSONBsqlFormat(sortOrderField.getCaseFieldId()));
+            Matcher matcher = SPECIAL_CHARS_PATTERN.matcher(sortOrderField.getCaseFieldId());
+            if (!matcher.find()) {
+                if (sortOrderField.isMetadata()) {
+                    sb.append(getMataFieldName(sortOrderField.getCaseFieldId()));
+                } else {
+                    sb.append(convertFieldNameToJSONBsqlFormat(sortOrderField.getCaseFieldId()));
+                }
+                sb.append(SPACE);
+                sb.append(sortOrderField.getDirection());
```

Comment:
We are appending an arbitrary String here - this should be an Enum, or it looks like you could use SortDirection.fromOptionalString

Reply:
Good point - done.